cmd/execute.go

```diff
@@ -164,6 +164,8 @@ var executeCmd = &cobra.Command{
 				exitCode = MultiErrorCode(cErr)
 			case MaxErrorsReachedError:
 				errMsg = cErr.Error()
+				// TODO set exit code when this has been ok'd
+				// exitCode = ExitCodeMaxErrorsReached
 				switch subErr := errors.Cause(cErr.SubError).(type) {
 				case *multierror.Error:
```

Comment:
Looks good. Will approve when this is uncommented and the error counting is removed.